docs/settings.md

```diff
@@ -0,0 +1,15 @@
+# Settings & Configuration
+
+## Settings
+
+### WAITINGLIST_SURVEY_SECRET
+
+Defaults to `SECRET_KEY`
+
+This is used for generating the hash for `pinax.waitinglist.models.SurveyInstance.code`.
+
+### DEFAULT_HTTP_PROTOCOL
```

Comment:
That was my first thought as well, but searching through [Settings] (https://docs.djangoproject.com/en/dev/ref/settings/) I wasn't able to find `DEFAULT_HTTP_PROTOCOL`.